src/controllers/client/booking.ts

```diff
@@ -4,18 +4,20 @@ import httpStatus from "http-status";
 import * as service from "@/services/client/booking";
 
 export interface Booking{
-    userId: number;
-    ticketInfo: TicketInfo;
-}
-
-interface TicketInfo {
   isOnline: boolean,
   hasHotel: boolean,
   price: number
 }
```

Comment:
Se essa interface for utilizada em outros arquivos, é recomendado que ela fique na pasta de `interfaces`.